docs/UserGuide.md

```diff
@@ -180,6 +180,8 @@ git config --global user.name "YOUR_AUTHOR_NAME”
 ```
 RepoSense expects the Git Author Name to be the same as author's GitHub username. If an author's `Git Author Name` is different from her `GitHub ID`, the `Git Author Name` needs to be specified in the standalone config file. If the author has more than one `Git Author Name`, multiple values can be entered too.
 
+> Note: As we use the OS terminal git command for analyse of repos, computer symbols (e.g. `"`, `!`, `/` etc.) inside the name will be replaced with a period `.` during analysis, which may reduce the accuracy of the analysis.
```

Comment:
I think a bit too much info. Perhaps just a `Note: using symbols such as... in your author name may reduce the accuracy of the analysis if 2 names are approximately similar`. will do

Reply:
Fixed